ddtrace/profiling/collector/memalloc.py

```diff
@@ -14,7 +14,7 @@
 from ddtrace.vendor import attr
 
 
-_MODULE_TOP_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+_MODULE_TOP_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))
```

Comment:
Does this mean the tracer is also going to be ignored when ignoring the profiler in memory profiles?

Reply:
Yes, and re-reading the ignore code I'm wondering if it's not going to be an issue since this code wraps customer/framework code.
I'm not sure how to fix that unless removing this `ignore_profiler` settings altogether.